build.sbt

```diff
@@ -340,6 +340,8 @@ lazy val walletTest = project
   .settings(
     name := "bitcoin-s-wallet-test",
     libraryDependencies ++= Deps.walletTest,
+    // To avoid deadlock issues with SQLite
+    Test / parallelExecution := false,
```

Comment:
Isn't this already applied here? https://github.com/bitcoin-s/bitcoin-s-core/blob/91b0de7320bdfe9833dc12c375c1937a7a70b4a0/build.sbt#L112

Reply:
That's only if we're on CI and macOS, as we had problems with RPC tests on macOS.